messenger.rst

```diff
@@ -1243,8 +1252,9 @@ for each bus looks like this:
 
 #. ``dispatch_after_current_bus``- see :doc:`/messenger/message-recorder`.
 
-#. ``failed_message_processing_middleware`` - sends failed messages to the
-   :ref:`failure transport <messenger-failure-transport>`.
+#. ``failed_message_processing_middleware`` - processes messages that are being
+   retried via the :ref:`failure transport <messenger-failure-transport>` to make
+   them properly function as if they were being received from their original transport.
 
 #. Your own collection of middleware_;
```

Comment:
Just spotted the inconsistent usage of `;` or `.` in this list. According to our "very strict" guideline on this, every list items should end with a semicolon except from the last one.